docs/configuration.md

```diff
@@ -111,6 +111,9 @@ The configuration file must be named `arduino-cli`, with the appropriate file ex
 - envfile
 - [INI]
 
+The configuration file [JSON schema][configuration-schema] can be used to independently validate the file content. This
```

Comment:
```suggestion
#### JSON schema

The configuration file [JSON schema][configuration-schema] can be used to independently validate the file content. This
```

Since this information is only vaguely related to the supported formats, I think it is better to put it under a dedicated section. This will make it easier for the reader to discover the information.